build.gradle

```diff
@@ -1,72 +1,76 @@
-buildscript {
-  repositories {
-    jcenter()
-  }
-}
-
 plugins {
     id 'java'
     id 'application'
     id 'base'
     id 'nebula.lint' version '9.3.4'
 }
-
-allprojects {
-  apply plugin: 'java'
-  apply plugin: 'base'
-  apply plugin: 'nebula.lint'
-  
-  repositories {
-    jcenter()
-  }
-
-  group = 'org.apereo.openequella.adminconsole'
-  version = artifactVersion
-
-  gradleLint.rules = ['all-dependency', 'dependency-parentheses', 'duplicate-dependency-class']
-
-  dependencies {
-    implementation 'org.slf4j:slf4j-api:1.7.26'
-    runtime 'org.slf4j:slf4j-simple:1.6.4'
-  }
-}
-
+/*
 subprojects {
   task sourcesJar(type: Jar) {
         from sourceSets.main.java
         archiveClassifier = 'sources'
     }
-}
+} */
 
-project(':common') {
-  dependencies {
-      implementation 'com.google.guava:guava:27.0.1-jre'
-      implementation 'com.fasterxml.jackson.core:jackson-databind:2.9.8'
-      implementation 'org.jasypt:jasypt:1.9.2'
-  }
+repositories {
+  jcenter()
 }
 
-project(':configurator') {
-  
-  mainClassName = 'org.apereo.openequella.adminconsole.configurator.Configurator'
-
-  apply plugin: 'application'
+group = 'org.apereo.openequella.adminconsole'
+version = artifactVersion
+mainClassName = 'org.apereo.openequella.adminconsole.launcher.ClientLauncher'
 
-  dependencies {
-    compile project(':common')
+apply plugin: 'java'
+apply plugin: 'base'
+apply plugin: 'nebula.lint'
+apply plugin: "application"
 
-    //implementation 'com.google.guava:guava:27.0.1-jre'
-    testImplementation 'junit:junit:4.12'
-  }
+// Gradlelint is broken: https://github.com/nebula-plugins/gradle-lint-plugin/issues/203
+gradleLint.rules = [/*'all-dependency', */'dependency-parentheses', 'duplicate-dependency-class']
 
-  // TODO: gradle complains if this isn't listed in both positions...
-  mainClassName = 'org.apereo.openequella.adminconsole.configurator.Configurator'
+dependencies {
+  implementation 'org.slf4j:slf4j-api:1.7.26'
+  implementation 'com.fasterxml.jackson.core:jackson-databind:2.9.8' 
+  implementation 'org.jasypt:jasypt:1.9.2'
+  runtimeOnly 'org.slf4j:slf4j-simple:1.6.4'
+  testImplementation "junit:junit:$junitVersion"
```

Comment:
If we've not tests, should we remove?

Reply:
Maybe.  Or I could write tests? :)  Basically just test reading and writing to the config file I guess.

Reply:
I'd never say no to some tests. ;)